src/Controller/Blockexplorer/NodeController.php

```diff
@@ -71,15 +66,6 @@ public function __construct(NodeRepositoryInterface $repository)
      */
     public function listAction(Request $request): Response
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this method "listAction" to simply inherit it. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=php%3AS1185)